### ISSUE TITLE
inserting "space" pngs

### ISSUE THREAD

**Issue by chanrxia** [orzubalsky/creative-computing-2018#20]
If we have to insert a new png for a space between a word, do we have to re number all the pngs from "manifesto_001" in order and on?

**Comment by danidls99** [comment]
Not necessarily! As long as at that point where you want the space, the image source is the file name of that png. so like for example <img src="images/(whatever you named the image of the space).png" alt="space" /_______________> 
We just named and numbered our pngs like that to make it easier to see all of our words in the correct order in Finder. I hope I made sense!
